CorpusSearch/ClientApp/src/routes/BitPlayer.tsx

```diff
@@ -63,23 +63,72 @@ function Video({vId="rLEBp8R1_XA"}) {
         setKtime(r_player.current.getCurrentTime())
         restart()
     }
-
     /* eslint-enable */
+    const alterTime = (timeString : string,timeFunc :React.Dispatch<React.SetStateAction<string>> ,chg : number) => {
+        const time_val = parseFloat(timeString) + chg
+        timeFunc(time_val.toFixed(2))
+    }
+
+    const alterStartTime = (chg : number) => {
+        alterTime(startTime,setStartTime,chg)
```

Comment:
I suspect you want the more simple syntax: `setStartTime(currentVal => currentVal + chg)`

Reply:
Sure, if we can get the previous thing working of making a number representation.

Reply:
Okay, I tried it, and it worked, did a compromise solution of inlining with the string
setStartTime( currentVal => (parseFloat(currentVal) + chg).toFixed(2) )